index.html

```diff
@@ -61,21 +65,17 @@
 							box-shadow: 0px 3px 2px rgba(0, 0, 0, 0.2);
 						}
 
-						paper-tabs::shadow #selectionBar {
+						paper-tabs #selectionBar {
```

Comment:
this will break under native shadow dom. Please use the css mixin (`--paper-tabs-selection-bar-color`)

https://elements.polymer-project.org/elements/paper-tabs